openupgrade_scripts/scripts/digest/14.0.1.1/post-migration.py

```diff
@@ -6,3 +6,10 @@
 @openupgrade.migrate()
 def migrate(env, version):
     openupgrade.load_data(env.cr, "digest", "14.0.1.1/noupdate_changes.xml")
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE digest_digest
+        SET mail_template_id = NULL
+        """,
+    )
```

Comment:
I think this is not needed, because in pre-migration you have already moved these values.